test/test-lint.cpp

```diff
@@ -1047,8 +1047,9 @@ TEST(test_lint, assign_to_mutable_variable_shadowing_immutable_variable) {
 TEST(test_lint, assign_to_immutable_variable) {
```

Comment:
NIT: This test needs to be renamed probably by "assign_to_immutable_const_variable" keeping it consistent with the naming paradigm we have used for assign_to_immutable_imported_variable.